fenix/app/src/main/java/org/mozilla/fenix/home/intent/ReEngagementIntentProcessor.kt

```diff
@@ -49,12 +46,12 @@ class ReEngagementIntentProcessor(
                         navController.nav(null, directions, options)
                     }
                     else -> {
-                        appStore.dispatch(AppAction.OpenTabInBrowser(BrowsingMode.Private))
```

Comment:
This was the only other place using this, so I think it makes sense to just remove it now that we have the ability to

Reply:
I think we need to also remove this in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/test/java/org/mozilla/fenix/home/intent/ReEngagementIntentProcessorTest.kt#89